recipes/bgfx/all/conanfile.py

```diff
@@ -191,27 +195,39 @@ def build(self):
 
             # Generate projects through genie
-            genieGen = f"{self._genie_extra} {compiler_and_os_to_genie[str(self.settings.os)]}"
+            genie_args = f"{self._genie_extra} {compiler_and_os_to_genie[str(self.settings.os)]}"
             if os_to_use_arch_config_suffix[str(self.settings.os)]:
-                genieGen += f"{gmake_arch_to_genie_suffix[str(self.settings.arch)]}"
-            genieGen += " gmake"
-            self.run(f"genie {genieGen}", cwd=self._bgfx_path)
+                if (self.settings.os == "Android"):
+                    genie_args += F"{gmake_android_arch_to_genie_suffix[str(self.settings.arch)]}"
+                else:
+                    genie_args += f"{gmake_arch_to_genie_suffix[str(self.settings.arch)]}"
+            genie_args += " gmake"
+            self.run(f"genie {genie_args}", cwd=self._bgfx_path)
 
             # Build project folder and path from given settings
-            projFolder = f"gmake-{gmake_os_to_proj[str(self.settings.os)]}"
-            if self.settings.os == "Windows" or compiler_str not in ["gcc", "apple-clang"]:
-                projFolder += f"-{compiler_str}" #mingw-gcc or mingw-clang for windows; -clang for linux (where gcc on linux has no extra)
+            proj_folder = f"gmake-{gmake_os_to_proj[str(self.settings.os)]}"
+            if self.settings.os == "Windows" or (compiler_str not in ["apple-clang"] and self.settings.os != "Android"):
+                proj_folder += f"-{compiler_str}" #mingw-gcc or mingw-clang for windows; -clang for linux (where gcc on linux has no extra)
             if os_to_use_arch_config_suffix[str(self.settings.os)]:
-                projFolder += gmake_arch_to_genie_suffix[str(self.settings.arch)]
-            proj_path = os.path.sep.join([self._bgfx_path, ".build", "projects", projFolder])
+                if (self.settings.os == "Android"):
+                    proj_folder += gmake_android_arch_to_genie_suffix[str(self.settings.arch)]
+                else:
+                    proj_folder += gmake_arch_to_genie_suffix[str(self.settings.arch)]
+            proj_path = os.path.sep.join([self._bgfx_path, ".build", "projects", proj_folder])
 
             # Build make args from settings
             conf = build_type_to_make_config[str(self.settings.build_type)]
             if os_to_use_make_config_suffix[str(self.settings.os)]:
                 conf += arch_to_make_config_suffix[str(self.settings.arch)]
             if self.settings.os == "Windows":
-                mingw = "MINGW=$MINGW_PREFIX"
-                proj_path = proj_path.replace("\\", "/") # Fix path for msys...
+                if "mingw-builds" in self.dependencies.build:
+                    # self.run("if [ ! -d /mingw64 ]; then mkdir /mingw64; fi")
+                    # self.run("pacman -Sy mingw-w64-x86_64-gcc --needed --noconfirm")
+                    # mingw = "MINGW=$MSYS_ROOT/mingw64"
```

Comment:
These lines seems unrelated or at least confusing, maybe should be removed.

I am not convinced about the check ``if "mingw-builds" in self.dependencies.build:`` for a specific dependency name in the dependencies, cause if someone uses another ``mingw_compiler`` package it won't work. Ideally the package itself could define the ``MINGW`` env-var or something like that.

But I also think we can move forward with it, and this can be discussed later.

Reply:
Sure, no problem, leave it as-is, I think we can figure this out later, after merge.